sched/scheduler/stateful_scheduler_test.go

```diff
@@ -46,7 +46,7 @@ func makeStatefulSchedulerDeps(deps *schedulerDeps) *statefulScheduler {
 		deps.sc,
 		deps.wf,
 		deps.config,
-		stats.NilStatsReceiver(),
+		stats.CurrentStatsReceiver,
```

Comment:
This is semantic only, for now CurrentStatsReceiver==NilStatsReceiver().  However, in all other test cases we use Current and not Nil; just something I noticed for consistency.